### DISCUSSION TITLE
TP3 - ¿La capacidad de las cadenas tiene que ser ingresada por el usuario?

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#107] [Preguntas y respuestas] [answered]
titulo

**Comment by martinvilu** [comment, marked as answer]
No, la capacidad es fija. El largo no.

Crea un par con capacidades diferentes, cosa de probar las diferentes situaciones.

**Reply by martinvilu** [reply]
C admite que las cadenas (y arreglos) tengan su tamaño definido por una variable, pero esto trae problemas. Y vamos a ver el tema puntualmente un poco más adelante.
